packer/plugin-getter/github/getter.go

```diff
@@ -0,0 +1,181 @@
+package github
```

Comment:
TODO: add gpg verification?

Reply:
So, the way I understood it from Paul is that the GPG verification happens when we crawl a plugin repo not on installation. Basically maintainers send us ( Hashicorp ) a GPG key, and when a new version is released: We check if the GPG signature of the checksum file is good, if so we allow the download of the official plugin.
But during installation only the checksum file should be verified.

Is this correct @sylviamoss ?

Reply:
The GPG is verified before pushing the plugin release to https://releases.hashicorp.com. Whatever is on our releases page, is verified and re-signed by HashiCorp. 

For Packer init, when downloading directly from GitHub, such verification won't happen.

Reply:
Thanks Moss 🙂